Automattic/wp-calypso
# Big Sky: Handle site_title, site_tagline

Solves https://github.com/Automattic/ai-services/issues/607

## Proposed Changes

* Get the new site_title and site_tagline parameters from the endpoint and pass them to the renderer endpoint
* Get the list of pages and pass them to the render endpoint

All this to fill out the navigation according to the generated data:
<img width="764" alt="Zrzut ekranu 2023-12-13 o 17 26 34" src="https://github.com/Automattic/wp-calypso/assets/3775068/a1398db0-09fb-4b0c-bd62-909b91fec87f">



## Testing Instructions

* Apply D131898-code
* Build this
* Go to http://calypso.localhost:3000/setup/ai-assembler
* Chose a site
* Write a prompt
* Observe content being filed out in headers

## Pre-merge Checklist
